### PR TITLE
Add openafs 1.6.18.3

### PR DESCRIPTION
**Pull request by alarcher**
Missing SMF manifest.

### CODE REVIEW

File: components/network/openafs/openafs.p5m [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+link path=usr/lib/libkopenafs.so target=libkopenafs.so.1.1
+link path=usr/lib/libkopenafs.so.1 target=libkopenafs.so.1.1
+file path=usr/lib/libkopenafs.so.1.1
+file path=usr/lib/liblwp.a
+file path=usr/lib/librx.a
+file path=usr/lib/librxkad.a
+file path=usr/lib/librxstat.a
+file path=usr/lib/libuafs.a
+file path=usr/lib/libubik.a
+file path=usr/lib/openafs/buserver
```
Comment (pyhalov) [inline components/network/openafs/openafs.p5m:284]: Do these files need special permissions (0555)?

File: components/network/openafs/openafs.p5m [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+<transform file path=kernel/drv/afs \
+    -> set action.hash usr/lib/openafs/libafs.o >
+file path=kernel/drv/afs
+
+<transform file path=kernel/drv/$(MACH64)/afs \
+    -> set action.hash usr/lib/openafs/libafs64.o >
+file path=kernel/drv/$(MACH64)/afs pkg.linted.userland.action001.2=true
+
+# Configuration
+file files/afs.rc     path=/etc/init.d/afs mode=0555
```
Comment (pyhalov) [inline components/network/openafs/openafs.p5m:35]: Can we convert init script to smf service?

File: components/network/openafs/Makefile [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+CONFIGURE_OPTIONS += --enable-kernel-module
+CONFIGURE_OPTIONS += --enable-namei-fileserver
+CONFIGURE_OPTIONS += --enable-bitmap-later
+CONFIGURE_OPTIONS += --with-krb5
+
+build:		$(BUILD_32)
+
+install:	$(INSTALL_32)
+
+test:		$(no_TESTS)
```
Comment (pyhalov) [inline components/network/openafs/Makefile]: NO_TESTS

File: components/network/openafs/Makefile [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
```
Comment (pyhalov) [inline components/network/openafs/Makefile]: Does package have any specific build requirements?

File: components/network/openafs/openafs.p5m (hunk 1 above)
Comment (alarcher) [reply components/network/openafs/openafs.p5m:284]: @pyhalov You are right but I have tested the client only for now.

File: components/network/openafs/openafs.p5m (hunk 2 above)
Comment (alarcher) [reply components/network/openafs/openafs.p5m:35]: @pyhalov I have SMF manifests from Mathias Feiler but I am not sure how they should be adapted.

File: components/network/openafs/Makefile (hunk 3 above)
Comment (alarcher) [reply components/network/openafs/Makefile]: @pyhalov Thanks

**Pull request closed without merging**